fontforgeexe/collabclientui.c

```diff
@@ -471,30 +471,31 @@ gpointer getSessionCallbacksObject()
 void collabclient_notifySessionJoining( cloneclient_t *cc, FontViewBase* fv )
 {
 #ifdef BUILD_COLLAB
-    
-    printf("AAA collabclient_notifySessionJoining() fv:%p\n", fv);
-    g_signal_emit_by_name( getSessionCallbacksObject(), "joining", fv );
-    
+    if (cc) {
```

Comment:
What was the occasion? Was this just happening when there was no network connection?